grid/face.py

```diff
@@ -0,0 +1,112 @@
+from itertools import chain
+from typing import Iterable, Mapping, List, NamedTuple, Optional, Type
+
+from model.base import ModelBase
+
+from .cell import Cell, Orientation
+from .geom import Address, Coordinate, Geometry, Position
+from .grid import Grid, Location, Pixel, Query, Selector
+
+PIXELS_PER_CELL: int = 8
+PIXELS_IN_ROW_TURNAROUND: int = 11
+
+
+class Panel(NamedTuple):
+    """
+    Panel represents an assembled panel in the pyramid frame.
+    """
+
+    geom: Geometry
+    start: Address
```

Comment:
Doesn't have to be this PR, but is this a good place to configure the dead pixel to skip on the one panel?

Reply:
yeah, we def could